[2023-12-08 01:46:20] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:20130)的加密handshake与协议handshake!

[2023-12-08 01:46:20] [info] server.go:602 dpchain/p2p/server.(*Server).listenLoop 接收到来自节点 127.0.0.1:63501的p2p连接请求

[2023-12-08 01:46:20] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:63501)的加密handshake与协议handshake!

[2023-12-08 01:46:20] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:63501)的加密handshake!

[2023-12-08 01:46:20] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:63501 

[2023-12-08 01:46:20] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn a226effe746e5dd9 127.0.0.1:63501 failed checkpoint posthandshake: Connected to self
[2023-12-08 01:46:20] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:20130)的加密handshake!

[2023-12-08 01:46:20] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:20130 

[2023-12-08 01:46:20] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn a226effe746e5dd9 127.0.0.1:20130 failed checkpoint posthandshake: Connected to self
[2023-12-08 01:46:52] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:20130)的加密handshake与协议handshake!

[2023-12-08 01:46:52] [info] server.go:602 dpchain/p2p/server.(*Server).listenLoop 接收到来自节点 127.0.0.1:63616的p2p连接请求

[2023-12-08 01:46:52] [info] server.go:616 dpchain/p2p/server.(*Server).setupConn p2pServer等待完成与目标节点(127.0.0.1:63616)的加密handshake与协议handshake!

[2023-12-08 01:46:52] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:63616)的加密handshake!

[2023-12-08 01:46:52] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:63616 

[2023-12-08 01:46:52] [info] server.go:636 dpchain/p2p/server.(*Server).setupConn p2pServer完成与目标节点(127.0.0.1:20130)的加密handshake!

[2023-12-08 01:46:52] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn a226effe746e5dd9 127.0.0.1:63616 failed checkpoint posthandshake: Connected to self
[2023-12-08 01:46:52] [info] server.go:643 dpchain/p2p/server.(*Server).setupConn 下面进行加密handshake的 checkpoint!!! ,127.0.0.1:20130 

[2023-12-08 01:46:52] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn a226effe746e5dd9 127.0.0.1:20130 failed checkpoint posthandshake: Connected to self
